charts/sentry-apps/ci/default-values.yaml

```diff
@@ -0,0 +1,6 @@
+---
+ingress:
+  hostname: sentry.example.org
+
+sentry:
+  postgresqlPassword: changeme
```

Comment:
We should ensure that `changeme` is never used.
```suggestion
  postgresqlPassword: 
```

Reply:
this is `ci`

Reply:
oh i see, sry